www/perl-lib/SMT/Patch.pm

```diff
@@ -380,11 +380,16 @@ sub save
         $description = substr($description, 0, SMT::Utils::MYSQL_TEXT_TYPE_SIZE);
     }
 
-    my $sth = $dbh->prepare($sql);
+    my $summary = $self->summary();
+    if (length($summary) > SMT::Utils::PATCH_SUMMARY_SIZE_CHARS) {
+        $summary = substr($summary, 0, SMT::Utils::MYSQL_TEXT_TYPE_SIZE);
+    }
+
+     my $sth = $dbh->prepare($sql);
```

Comment:
Is there a specific reason for indenting this line by one?

Reply:
Unspecific fat fingeredness.